doc/ref_cert/README.md

```diff
@@ -1,5 +1,43 @@
 [<< Back](../)
 
+# CNTT Reference Conformance (RC) Requirements and Testing Principles
+
+The objective each RC workstream is to provide an automated mechanism to measure either a network function (NF) or a cloud infrastructure with a standard set of requirements defined by the associated Reference Architecture (RA).  Through this measurement, a provider of either NFs or cloud infrastructure will be able to test their conformance to the RA.  This will ease the integration of NFs into operator environments that host compatible cloud infrastructures, thereby reducing cost, complexity, and time of integration.
+
+The overall workstream requires the close coordination of the following:
+
+**Requirements** - The agreed upon rules and recommendations to which a compliant NF or cloud infrastructure must adhere. All requirements will be hosted and maintianed in the RA.
+**Tests** - The verification mechanism that determines that a given NF or cloud infrastructure complies with one or more requirements.
+**Conformance Specifications** - The definition of the requirements, tests, and circumstances (test case integration, etc.) that must be met to be deemed conformant.
+
+If there is no clear traceability and strong links between these 3 components, then it becomes difficult to determine if a NF or cloud infrastructure is compliant. With this in mind, below are the set of recommended principles for each of the three components to follow. Adherence to these principles will provide the following:
+
+* Enable clear progress tracking and linkage between independent projects (i.e. know what has and hasn't been covered, and track changes over time)
+* Help users better understand mandatory requirements
+* Provide a stable set of point-in-time requirements and tests to achieve conformance.  
+* Reduce ambiguity in testing, requirements, and conformance
+
+
+## Requirement Principles
```

Comment:
These Requirements Principles need to be at the CNTT root level as RM and RA's have to conform to them

Reply:
Where do you suggest they go then? Also do you think we should remove this section entirely or not?

Reply:
@xmulligan One possible place would be the [Principles](https://github.com/cntt-n/CNTT/blob/master/doc/tech/README.md#2-principles) section in tech/README. Currently, this section consists of [2.1 Overall Principles](https://github.com/cntt-n/CNTT/blob/master/doc/tech/README.md#21-overall-principles) and [2.2 Architectural Principles](https://github.com/cntt-n/CNTT/blob/master/doc/tech/README.md#22-architectural-principles).

Reply:
The links above are broken and I can't seem to find which parts you are pointing to. Can you repost them?

Reply:
@xmulligan Sorry, content got reorganised:
[Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#principles): https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#principles

[Overall Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#overall-principles): https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#overall-principles

[Architectural Principles](https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#architectural-principles): https://github.com/cntt-n/CNTT/blob/master/doc/common/chapter00.md#architectural-principles

Reply:
Thanks moved to #1797